packages/panels/resources/views/components/tenant-menu.blade.php

```diff
@@ -31,6 +31,7 @@
 <x-filament::dropdown
     placement="bottom-start"
     teleport
+    size="{availableWidth: 300, availableHeight: 300}"
```

Comment:
Can we have individual `availableWidth` and `availableHeight` props instead of this JSON please?

```suggestion
    size="{availableWidth: 300, availableHeight: 300}"
```

Reply:
We can. I was just following the settings convention for the plugin that uses object based configs.

Reply:
Yeah, I just think it's better for autocomplete that way, we can json_encode the options to form that config

Reply:
No worries. Also not sure why the checks are failing. Didn't modify any php files. 😂

Reply:
Maybe they'll be fine when they rerun, weird